api/src/services/opportunities_v1/get_opportunity.py

```diff
@@ -31,7 +34,30 @@ def _fetch_opportunity(
 
 
 def get_opportunity(db_session: db.Session, opportunity_id: int) -> Opportunity:
-    return _fetch_opportunity(db_session, opportunity_id, load_all_opportunity_summaries=False)
+    opportunity = _fetch_opportunity(
+        db_session, opportunity_id, load_all_opportunity_summaries=False
+    )
+    opportunity_copy = copy.deepcopy(opportunity)
+
+    s3_config = S3Config()
+    s3_config.s3_endpoint_url = "http://localstack:4566"
```

Comment:
This should be getting loaded from env vars properly, were you hitting an issue where it wasn't?